pkg/ext/stores/tokens/tokens.go

```diff
@@ -274,6 +274,7 @@ func (t *Store) Create(
 }
 
 // DeleteCollection implements [rest.CollectionDeleter]
+// Calls on deleteCore.
```

Comment:
```suggestion
// Calls deleteCore.
```

Reply:
Not sure that such comment even needed. It's self evident.

Reply:
Removed